pycode/memilio-epidata/memilio/epidata/getPopulationData.py

```diff
@@ -294,7 +331,8 @@ def get_population_data(read_data=dd.defaultDict['read_data'],
     @param password Password to sign in at regionalstatistik.de.
```

Comment:
```suggestion
    @param password Password to sign in at regionalstatistik.de. Can be read from CredentialsRegio.ini if existent. 
```
I think here it should be mentioned that the password can be read from .ini file/ stored in .ini file. Same for username.